src/peft/peft_model.py

```diff
@@ -197,7 +197,10 @@ def save_pretrained(
             peft_config = self.peft_config[adapter_name]
             # save only the trainable weights
             output_state_dict = get_peft_model_state_dict(
-                self, state_dict=kwargs.get("state_dict", None), adapter_name=adapter_name
+                self,
+                state_dict=kwargs.get("state_dict", None),
+                adapter_name=adapter_name,
+                base_model_layers_to_save=kwargs.get("base_model_layers_to_save", None),
```

Comment:
Instead of doing `kwargs.get`, can we add `base_model_layers_to_save` to the signature of `save_pretrained` and also add a docstring for it (same as for `get_peft_model_state_dict`)?